decidim-budgets/app/controllers/decidim/budgets/admin/projects_controller.rb

```diff
@@ -217,9 +219,11 @@ def sample_project
         end
 
         def project
-          @project ||= projects.find(params[:id])
+          @project ||= filtered_collection.find_by(id: params[:id])
```

Comment:
Why not taking into account the not_trashed state ?
```suggestion
          @project ||= projects.find(params[:id])
```